dropgz/pkg/embed/payload.go

```diff
@@ -86,6 +87,20 @@ func deploy(src, dest string) error {
 		return err
 	}
 	defer rc.Close()
+	_, err = os.Stat(dest)
+	if err != nil {
+		if !os.IsNotExist(err) {
+			return err
+		}
+	} else {
+		oldDest := dest + oldFileSuffix
+		if err = os.RemoveAll(oldDest); err != nil {
+			return errors.Wrapf(err, "failed to remove the %s", oldDest)
+		}
```

Comment:
lets remove this and rename should take care of it

Reply:
Done